src/main/resources/templates/404.html

```diff
@@ -0,0 +1,16 @@
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta charset="UTF-8"/>
+  <title>Not Found</title>
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz" rel="stylesheet" type="text/css">
+  <link href="https://faforever.com/styles/css/site.min.css" rel="stylesheet" type="text/css"/>
+  <link rel="stylesheet" th:href="@{/css/style.css}" type="text/css"/>
+</head>
+<body>
+<div class="background hero"></div>
+<h1 class="primary-color" style="position: fixed; top: 30%; left: 20%;">You will get what you have asked for.
+  Nothing...</h1>
```

Comment:
I wish we had something more like `supreme commander` information.

Like:
```
There is nothing left, commander.
```

Or
```
We couldn't find the page for you, commander.
```

Or:
```
Dostya: No vodka found in the trunk...
```
etc...